### PR TITLE
Relax omniauth oauth dependency

### PR DESCRIPTION
**Pull request by franciscoj**
### :tophat: What?

Relax omniauth dependency since this allow upgraing to `1.x`

* Remove `Gemfile.lock` since it is not supossed to be on version control
* Remove unneeded dev dependencies

### CODE REVIEW

**Comment by saimonmoore** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Change Log
+
+## [Unreleased]
+
+### Changed
+- Relaxed `omniauth` dependency to so that it fits `1.x`.
```
Comment (sauloperez) [inline CHANGELOG.md]: dependency to?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Saner development dependencies.
```
Comment (sauloperez) [inline CHANGELOG.md]: Saner?

**Comment by sauloperez** [comment]
:postal_horn: :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (franciscoj) [inline CHANGELOG.md]: More sane? is better? (I'm a bit of a word inventer, sorry)

Comment (sauloperez) [inline CHANGELOG.md]: Sorry, I thought it was an invention but seems to be right :trollface:

Comment (franciscoj) [inline CHANGELOG.md]: LOL no problem. I still feel like Shakespeare.

**Comment by ivanilves** [comment]
:+1:

**Pull request merged**